docker-compose.yml

```
@@ -275,7 +283,11 @@ services:
     profiles:
       - ingestion_server
       - api
-    build: ./ingestion_server/
+    build:
+      context: ./ingestion_server/
+      target: ing
+      args: # Automatically inferred from env vars, unless specified
+        - ING_PY_VERSION
```

nit: `ingestion` would be preferable over `ing` for more clarity.
```suggestion
      target: ingestion
      args: # Automatically inferred from env vars, unless specified
        - INGESTION_PY_VERSION
```

---

While I love the three letter codes we have now, 'api', 'ing' (ingestion server), 'cat' and 'app' (frontend) I agree that the `INGESTION` is much clearer than `ING`. Updated the name in 91dbecb.